engine/table/src/main/java/io/deephaven/engine/table/impl/partitioned/LongConstantColumn.java

```diff
@@ -44,7 +46,10 @@ public List<String> initInputs(
     }
 
     @Override
-    public List<String> initDef(@NotNull final Map<String, ColumnDefinition<?>> columnDefinitionMap) {
+    public List<String> initDef(
+            @NotNull final Map<String, ColumnDefinition<?>> columnDefinitionMap,
+            @NotNull final Supplier<Map<String, Object>> queryScopeVariables,
+            @NotNull final QueryCompilerRequestProcessor compilationRequestProcessor) {
         return getColumns();
     }
```

Comment:
I'm inclined to think that you would have less code to write if you defaulted the long-form initDef to the short-form.